### PR TITLE
ST for labels verification

### PR DESCRIPTION
**Pull request by asorokhtey**
### Type of change

- New System test

### Description

ST to verify that all resources have proper labels

### Checklist

- [ ] Update/write design documentation in `./design`
- [X] Write tests
- [X] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [X] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java [hunk 1]
```diff
@@ -99,16 +99,29 @@
...
+        oc.newApp(appName, map("CLUSTER_NAME", clusterName));
         oc.waitForStatefulSet(zookeeperClusterName(clusterName), 3);
         oc.waitForStatefulSet(kafkaClusterName(clusterName), 3);
         oc.waitForDeployment(entityOperatorDeploymentName(clusterName), 1);
 
         //Testing docker images
         testDockerImagesForKafkaCluster(clusterName, 3, 3, false);
 
+        //Testing labels
+        verifyLabelsOnCOPod(NAMESPACE);
```
Comment (Frawless) [inline systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java]: These all methods could be called from one method which will contains all of them. Test itself will be much cleaner I think.

File: systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java [hunk 2]
```diff
@@ -1089,6 +1089,257 @@ void waitForPodDeletion(String namespace, String podName) {
             () -> CLIENT.pods().inNamespace(namespace).withName(podName).get() == null);
     }
 
+    void verifyLabelsOnZkPods(String namespace, String clusterName, int zkPods, String appName) {
```
Comment (Frawless) [inline systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java]: @asorokhtey You can definitely parameterize similar methods. I can see there a lot of methods which are doing basically same thing and differs only in strings for pod names. Guess you can have 1 method for pods check, 1 for services, 1 for config maps and so on

**Comment by scholzj** [comment]
@Frawless @serrss What should we do with this PR? You approved it, but it seems that the tests never passed and it was not merged. Should this be reviewed? Should this be closed?

**Comment by Frawless** [comment]
@scholzj I will take a look on it this week. Guess we just need rebase and run tests.

**Comment by serrss** [comment]
@Frawless rebased a branch on the master to solve conflicts and adapted code to use Kubernetes client

**Comment by serrss** [comment]
@strimzi-ci run tests testcase=KafkaST#testDeployKafkaClusterViaTemplate,MirrorMakerST#testMirrorMaker,ConnectST#testDeployUndeploy

**Comment by serrss** [comment]
@strimzi-ci run tests

**Comment by serrss** [comment]
@strimzi-ci run tests

**Comment by serrss** [comment]
@strimzi-ci run tests testcase=ConnectST,KafkaST

**Pull request merged**